packages-internal/docs-utils/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/internal-docs-utils",
-  "version": "1.0.5",
+  "version": "1.0.6",
```

Comment:
Should we update the other internal packages? We have two at 1.0.2 and two at 1.0.6. I think it would be easier for the release manager to keep them all with the same version, and update them everytime.

Reply:
Their versions won't necessarily match (because of breaking changes). During the weekly release it should be safe to bump the patch version on each one.

Reply:
So should I bump all patch versions on all? That would be

- `1.0.5` => `1.0.6`
- `1.0.2` => `1.0.3`

Reply:
All packages that have changes.